airflow/override-values.yaml

```diff
@@ -6,41 +6,28 @@ nodeSelector:
 executor: KubernetesExecutor
 
 config:
+  smtp:
+    smtp_host: smtp.gmail.com
+    smtp_starttls: True
+    smtp_ssl: False
+    smtp_port: 587
+    smtp_mail_from: cbioportalpipelines@gmail.com
   core:
-    # This changed in newer versions of Airflow
-    # Compare: https://airflow.apache.org/docs/apache-airflow/2.7.1/configurations-ref.html#core
-    # and https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#core
     allowed_deserialization_classes_regexp: airflow\..* .*\..*StudyBuilder
   webserver:
     base_url: https://airflow.cbioportal.dev.aws.mskcc.org
-      #  metrics:
-      #    statsd_on: true
-      # statsd_host: 
-      #statsd_port: 8125
-      #statsd_prefix: airflow
-  # Starting in Airflow >=2.5.0, KubernetesExecutor is configured with
-  # [kubernetes_executor] and not [kubernetes]
-  #kubernetes_executor:
-  #  pod_template_file: /opt/airflow/pod-template.yaml
-  #  worker_container_repository: jamesko/airflow-worker-nci
-  #  worker_container_tag: 0.1
   kubernetes_executor:
     delete_worker_pods: True
     delete_worker_pods_on_failure: True
 
 images:
   airflow:
+    # TODO: the Docker image should live under the cbioportal/ namespace
     repository: jamesko/airflow
     tag: "1.2"
     pullPolicy: Always
-  #pod_template:
-  #  repository: jamesko/airflow-init
-  #  tag: "0.2"
-  #  pullPolicy: Always
 
 env:
-  - name: "AIRFLOW_VAR_CDM_UTILS_PATH"
-    value: "/opt/airflow/git_repos/cdm-utilities"
   - name: "PYTHONPATH"
     value: "/opt/airflow/dags/repo/nci-crdc-pipeline"
```

Comment:
Wonder if it's possible to remove this (either now or eventually) since we want things to be project-agnostic

Reply:
Would be a good idea to get rid of this if possible but I don't think it needs to be removed for this PR. Something for you to look into in the future